redhat-img/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+tag="red:v1"
```

Comment:
This is not only the tag but also the image name. Should it be some name like kubeturbo:rhel7?

Reply:
yea, it should be `kubeturbo:<tag>`.